### ISSUE TITLE
[CLI]: tensorboardX is unexpected keyword

### ISSUE THREAD

**Issue by Msameim181** [wandb/client#3571]
### Describe the bug

When I want to use TensorBoard to in WanB I got this:

```
wandb.tensorboard.patch(root_logdir="./tensorboard", tensorboardX=True)
```

```
    wandb.tensorboard.patch(root_logdir="./tensorboard", tensorboardX=True)
TypeError: patch() got an unexpected keyword argument 'tensorboardX'
```

In reference:
![image](https://user-images.githubusercontent.com/25700939/165082915-cfd8eb5e-f565-4674-a10b-cb8412d7a64b.png)
Link: https://docs.wandb.ai/guides/integrations/tensorboard#syncing-previous-tensorboard-runs

### Environment

WandB version:  0.12.14

OS: Windows 10

Python version: 3.9.0

Versions of relevant libraries: 
torch=1.11.0+cu113
tensorboardX=2.5

**Comment by ash0ts** [comment]
Hello @Msameim181,

I apologize for the confusion here. We recently updated our Tensorboard integration and changed the variable of `tensorboardX` to `tensorboard_x`. You would need to call `wandb.tensorboard.patch(root_logdir="./tensorboard", tensorboard_x=True)`. We will make a doc update to reflect this. Thank you for bringing this to our attention.

**Comment by Msameim181** [comment]
Hello to you Sir,

Well, Thank you for your response on such short notice. I will check that. But in the meantime, This [Question](https://github.com/wandb/client/issues/3572#issue-1214452943) is still a mystery for me.

**Comment by ash0ts** [comment]
Hi @Msameim181 ,

Just wanted to check in to see if the `tensorboard_x` helped

**Comment by Msameim181** [comment]
Yes, I checked that now. That Error has been solved, But this issue #3572 still stands.

I tried ```tensorboardX``` as a library too.  The ```torch.utils.tensorboard``` or ```tensorboardX``` was working well in local tensorboard server. But there was nothing on WandB Server.

I'm using:
```
wandb.tensorboard.patch(root_logdir="./tensorboard", tensorboard_x=True)
```

and others like #3572 ,
### WandB Server (Run Page):
![image](https://user-images.githubusercontent.com/25700939/165923859-b8404571-1b29-47c0-aa8b-e55550c6bbba.png)

### TensorBoard (local Server with ```$ tensorboard --logdir=tensorboard```):
(This is a run with tensorboardX)
![image](https://user-images.githubusercontent.com/25700939/165923721-e265b43e-0c92-4086-8c7d-eb1cc5903448.png)
